services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceHelixAdmin.java

```diff
@@ -4759,9 +4760,18 @@ public void deleteHelixResource(String clusterName, String kafkaTopic) {
     LOGGER.info("Successfully dropped the resource: {} for cluster: {}", kafkaTopic, clusterName);
 
     List<String> instances = getStorageNodes(clusterName);
+    Map<String, List<String>> disabledPartitions;
     for (String instance: instances) {
-      Map<String, List<String>> disabledPartitions =
-          getHelixAdminClient().getDisabledPartitionsMap(clusterName, instance);
+      try {
+        disabledPartitions = getHelixAdminClient().getDisabledPartitionsMap(clusterName, instance);
+      } catch (HelixException helixException) {
+        LOGGER.warn(
```

Comment:
Won't this flood the logs? Can we rate-limit this?

Reply:
Its should not be that many,  it can happen only during deleting old version if some instance configs are not in sync.

Reply:
Yes for each new push it will print once for each out of sync configs.